stack.yaml

```diff
@@ -6,28 +6,22 @@ resolver: nightly-2016-05-29
 # Local packages, usually specified by relative directory name
 packages:
 - '.'
+# timezone-series with time-1.6 support
+- location:
+    git: git@github.com:ryantrinkle/timezone-series.git
+    commit: f8dece8c016db6476e2bb0d4f972769a76f6ff40
+  extra-dep: true
+
+# For timezone-olson
+allow-newer: true
 
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
 extra-deps:
-- bindings-posix-1.2.6
+- thyme-0.3.5.5
```

Comment:
Quick question: why did you need this? That's the version for thyme in the nightly, or am I seeing something wrong?